### PR TITLE
Rename bn_bias_params + add kwargs in core

### PR DESCRIPTION
**Pull request by KushajveerSingh**
Add `use_kwargs_dict` in the following functions
1. tensors (dtype=None, device=None, requires_grad=False, pin_memory=False)
2. BatchNorm (eps=1e-5, momentum=0.1, affine=True, track_running_stats=True)
3. InstanceNorm (eps=1e-5, momentum=0.1, track_running_stats=True)
4. ConvLayer (dilation=1, groups=1, padding_mode='zeros')
5. CrossEntropyLossFlat (weight=None, ignore_index=-100, reduction='mean')
6. BCEWithLogitsLossFlat (weight=None, reduction='mean', pos_weight=None)
7. BCELossFlat (weight=None, reduction='mean')
8. MSELossFlat (reduction='mean')
9. L1LossFlat (reduction='mean')
10. LabelSmoothingCrossEntropyFlat (eps=0.1, reduction='mean')

Renamed `bn_bias_params` to `norm_bias_params`.

`L1LossFlat` documentation stated "Same as `nn.MSELoss`". Fixed the
typo.

### CODE REVIEW

**Comment by jph00** [comment]
Thanks for the PR! Can you explain why this one?: "Renamed bn_bias_params to norm_bias_params"

**Comment by KushajveerSingh** [comment]
There was a TODO to change the name.

```python
norm_types = (nn.BatchNorm1d, nn.BatchNorm2d, nn.BatchNorm3d, nn.InstanceNorm1d, nn.InstanceNorm2d, nn.InstanceNorm3d)

# in the function
if isinstance(m, norm_types): return L(m.parameters())
```

`bn_bias_params` is a general function, so the name change.

**Comment by jph00** [comment]
Thanks!

**Comment by jph00** [comment]
FYI there's some places you could have used `delegates` to do this more easily, so I'll make those changes now.

**Comment by KushajveerSingh** [comment]
There are some redundant arguments also, I think in the losses.
